angular/projects/common/src/app/models/document.ts

```diff
@@ -6,20 +6,17 @@
  */
 export class Document {
   _id: string;
-  _record: string;
   _addedBy: string;
-  documentFileName: string;
-  displayName: string;
-  internalURL: string;
-  internalMime: string;
+  fileName: string;
+  key: string;
+  url: string;
+  upfile: File;
```

Comment:
This value isn't used anywhere here.  Should it be? Or can it be removed? (Unless it gets used in a later update).

Reply:
It is used. but only for the front end. Its used to hold the binary. we dont actually want to send this over to the api

Reply:
Maybe just add a comment then.
The linter might complain as well, so we may need to add a lint-ignore comment.